packages/evolution-common/src/utils/ConstructorUtils.ts

```diff
@@ -14,17 +14,25 @@ export class ConstructorUtils {
      * @template T - The type of the params object.
      * @param params - The parameters object containing the attribute values.
      * @param attributeNames - The array of attributes names.
+     * @param attributeWithComposedAttributes - The array of attributes that have composed attributes. composed attributes are dealt with separately
```

Comment:
What is a composed attribute and why dealt with separately?
Je pense avoir mélangé "composed" et "custom" dans mon review plus haut...

Reply:
composed = separate object, using another class, validated separately. custom = custom attribute, saved in a json field in the db.